### DISCUSSION TITLE
Halmet wifi password

### DISCUSSION THREAD

**Discussion by pmitham** [hatlabs/discussions#91] [Q&A]
After uploading the example firmware to the Halmet,  I see a SSID called " Configure Halmet"  but I can't find in the src code what the password for this SSID is.  once connected will I be able to configure values for the various analogue and digial inputs?  Unfortunately its not clear at all how to configure this board for various analogue and digital sensors.

**Comment by mairas** [comment]
The default password is `thisisfine`. See here for more information:

https://signalk.org/SensESP/pages/getting_started/#wifi-configuration
